### DISCUSSION TITLE
Multiline placeholder text only showing one line when iOS keyboard open after clearing typed text

### DISCUSSION THREAD

**Discussion by FullStackJohnMack** [tailwindlabs/tailwindcss#4877] [Help]
### What version of Tailwind CSS are you using?

2.2.4

### What build tool (or framework if it abstracts the build tool) are you using?

create-react-app using postcss7

### What version of Node.js are you using?

14.16.0

### What browser are you using?

Chrome and Safari

### What operating system are you using?

iOS 14.3

### Reproduction repository

https://github.com/FullStackJohnMack/test-textarea-tailwind

### Describe your issue

I have a multiline text area with wrapping placeholder text wrapped in `<div className="flex flex-col">`. It should be noted that **this does not happen when just `flex` is used**. The bug (or my ignorance) presents itself over the following four steps:

1. The placeholder text shows up correctly
2. I type something into the text area and the placeholder text correctly disappears
3. I delete all my typed text **and leave the keyboard open**
4. I now only see the first line of the long placeholder text until I close the keyboard

I would expect to see the full placeholder text instead of just the first line. This forces our user to close the keyboard, reopen it, and then start typing again if they wanted to read the full placeholder text again.

This issue does not happen on Chrome's devtools when previewing my project... only on my actual mobile devices.

**Comment by adamwathan** [comment]
Hey! Do you see the same result if you just use totally vanilla CSS without using Tailwind at all? If so this is unfortunately just a browser CSS issue that we won't be able to fix.

**Comment by adamwathan** [comment]
Going to convert to a discussion in case anyone has suggestions for workarounds for you — I expect this same thing happens with vanilla CSS given how Tailwind is implemented so probably nothing we can do here I'm afraid :(
